idea/src/org/jetbrains/kotlin/idea/inspections/collections/RenameUselessCallFix.kt

```diff
@@ -35,6 +34,25 @@ class RenameUselessCallFix(val newName: String) : LocalQuickFix {
             val selectorCallExpression = it.selectorExpression as? KtCallExpression
             val calleeExpression = selectorCallExpression?.calleeExpression ?: return
             calleeExpression.replaced(factory.createExpression(newName))
+            selectorCallExpression.applyFix(factory, calleeExpression.text, newName)
+        }
+    }
+
+    private fun KtCallExpression.applyFix(factory: KtPsiFactory, calleeName: String, newName: String) {
+        val lambdaExpression = lambdaArguments.firstOrNull()?.getLambdaExpression() ?: return
+        val bodyExpression = lambdaExpression.bodyExpression ?: return
+
+        bodyExpression.statements.forEach {
```

Comment:
You should probably use `bodyExpression.forEachDescendantOfType<KtReturnExpression>`, otherwise you can miss multiple return descendants of the same statement